Code/Editor/TrackView/TrackViewNodes.cpp

```diff
@@ -1320,7 +1320,13 @@ void CTrackViewNodesCtrl::OnNMRclick(QPoint point)
                 }
             } while (retryRename);
 
-            if (!newName.isEmpty())
+            if(!GetNodeRecord(animNode2)) {
+                QMessageBox::warning(
+                    this,
+                    tr("Entity does not exists"),
```

Comment:
Maybe be an opportunity to change the text to be "Entity does not exist" (plural -> singular)